guides/common/modules/snip_step-parameter-ak.adoc

```diff
@@ -0,0 +1,3 @@
+. Click the *Parameters* tab.
+Add the `kt_activation_keys` parameter and enter the name of the activation key as the value.
+You can also enter a comma-separated list of multiple activation keys.
```

Comment:
@stejskalleos Is this sentence accurate? It was a guess :)

Reply:
The thing is that on some pages, like the host group, there is a UI for selecting activation keys, so you don't have to write them manually. But for some pages, AFAIK, the input is just a text field, where you have to write them manually.

cc @jeremylenz

Reply:
There's an Activation Keys tab for hostgroups, but not for creating or editing hosts. The Activation Keys tab is nice UI for adding the parameter, so the end result is always the same. The comma-separated list will work fine. :)